### PR TITLE
replication: add pvcRef to volumegroupreplication

### PR DESCRIPTION
**Pull request by Madhu-1**
adding the pvcRef to the volumegroupreplication to give a representation of the pvc's which are part of the volumegroupreplication.

### CODE REVIEW

File: apis/replication.storage/v1alpha1/volumegroupreplication_types.go [hunk 1]
```diff
@@ -72,6 +73,10 @@ type VolumeGroupReplicationSource struct {
 // VolumeGroupReplicationStatus defines the observed state of VolumeGroupReplication
 type VolumeGroupReplicationStatus struct {
 	VolumeReplicationStatus `json:",inline"`
+	// PersistentVolumeClaimsRefList is the list of of PVC for the volume group replication
```
Comment (iPraveenParihar) [inline apis/replication.storage/v1alpha1/volumegroupreplication_types.go:76]:
```suggestion
	// PersistentVolumeClaimsRefList is the list of of PVCs for the volume group replication.
```

File: apis/replication.storage/v1alpha1/volumegroupreplication_types.go [hunk 2]
```diff
@@ -72,6 +73,10 @@ type VolumeGroupReplicationSource struct {
...
+	// The maximum number of allowed PVC in the group is 100.
```
Comment (iPraveenParihar) [inline apis/replication.storage/v1alpha1/volumegroupreplication_types.go:77]:
```suggestion
	// The maximum number of allowed PVCs in the group is 100.
```

**Review by iPraveenParihar — COMMENTED** [review]
nit

File: apis/replication.storage/v1alpha1/volumegroupreplication_types.go [hunk 3]
```diff
@@ -72,6 +73,10 @@ type VolumeGroupReplicationSource struct {
 // VolumeGroupReplicationStatus defines the observed state of VolumeGroupReplication
 type VolumeGroupReplicationStatus struct {
 	VolumeReplicationStatus `json:",inline"`
+	// PersistentVolumeClaimsRefList is the list of of PVCs for the volume group replication.
```
Comment (yati1998) [inline apis/replication.storage/v1alpha1/volumegroupreplication_types.go:76]: `the list of of PVCs ` -> `the list of PVCs `

**Review by yati1998 — COMMENTED** [review]
just a small nit

**Review by yati1998 — APPROVED** [review]

File: apis/replication.storage/v1alpha1/volumegroupreplication_types.go [hunk 4]
```diff
@@ -72,6 +73,10 @@ type VolumeGroupReplicationSource struct {
 // VolumeGroupReplicationStatus defines the observed state of VolumeGroupReplication
 type VolumeGroupReplicationStatus struct {
 	VolumeReplicationStatus `json:",inline"`
+	// PersistentVolumeClaimsRefList is the list of PVCs for the volume group replication.
+	// The maximum number of allowed PVCs in the group is 100.
```
Comment (iPraveenParihar) [inline apis/replication.storage/v1alpha1/volumegroupreplication_types.go:77]: @Madhu-1, What is the rationale for setting the maximum number of PVCs in a group to 100?

Comment (Madhu-1) [reply apis/replication.storage/v1alpha1/volumegroupreplication_types.go:77]: This is the support max number of PVC in a group. we already have this in VGRC for PV object

**Review by iPraveenParihar — APPROVED** [review]

**Review by iPraveenParihar — APPROVED** [review]

**Pull request merged**